code/game/machinery/computer/medical.dm

```diff
@@ -140,6 +140,20 @@
 	onclose(user, "med_rec")
 	return
 
+
+/obj/machinery/computer/med_data/attackby(var/obj/item/O, var/mob/user)
+	if (authenticated && istype(O, /obj/item/weapon/photo/id) && (screen == 4.0) && active1)
+		var/obj/item/weapon/photo/id/photo_id = O
+		if (photo_id.four_sides)
+			if (alert("Do you want to update the records with this ID photo?",,"Yes","No") == "Yes")
+				if ((screen == 4.0) && active1)
```

Comment:
probably not majorly noticeable but there's no proximity or incapacitated check after the alert meaning you can walk away and update someone's picture after dying